### ISSUE TITLE
Logic Pro X renamed to just Logic Pro

### ISSUE THREAD

**Issue by jordito** [davidteren/lpx_links#12]
Hi David,

Back in 2020 the Logic Pro X app was renamed to just "Logic Pro". 

Can you please update line 10 of lpx_links/lib/**file_helpers.rb** from 
```
'/Applications/Logic Pro X.app/Contents/Resources'
```
 to something like 
```
`/Applications/Logic Pro\*.app/Contents/Resources`
```
?

Thanks.

**Comment by QuakersOat** [comment]
Jordito, this is unnecessary and actually would break things for people on older versions. Apple was smart enough to automatically create a symlink for backwards compatibility, I just checked it out.

**Comment by davidteren** [comment]
Thanks for checking it out @QuakersOat.

@jordito does the current path break for for you?

**Comment by jordito** [comment]
Hi Guys,

@QuakersOat thanks for the feedback. I realized that what I suggested above is not even valid Ruby (time to learn Ruby I guess), but my intention was precisely to suggest something that would allow the data for the links to be extracted regardless of whether the app was called "Logic Pro" or "Logic Pro X". For that the following works but it slows the script a bit so there's probably a better way:

```
"/Applications/Logic Pro#{'' || ' X'}.app/Contents/Resources"
```

@davidteren , the day I opened this issue, executing your **lpx_links.rb** script did not work at all for me on macOS 11.6.8 with Logic Pro 10.7.4. All I got was the following output in my terminal:
 
![LogicPro](https://user-images.githubusercontent.com/18441852/189216406-f1dc53a0-db04-4695-8425-7d2ccb29b689.png)

I've tried again recently and after a few tries it suddenly started working (not sure why), but I still get this output in my terminal apparently coming from your **plist_file_name** function with the embedded shell commands:
```
sh: line 0: cd: /Applications/Logic Pro X.app/Contents/Resources: No such file or directory
``` 
Do either of you confirm this behavior?

@QuakersOat, I'm curious about that symlink you mentioned. Where are you seeing it?

Thanks.
